packages/features/ee/organizations/components/OrganizationAvatar.tsx

```diff
@@ -0,0 +1,41 @@
+import classNames from "@calcom/lib/classNames";
+import { Avatar } from "@calcom/ui";
+import type { AvatarProps } from "@calcom/ui";
+
+type OrganizationAvatarProps = Pick<
+  AvatarProps,
+  "size" | "imageSrc" | "alt" | "indicator" | "gravatarFallbackMd5"
+> & {
+  organizationSlug?: string | null;
+};
+
+const OrganizationAvatar = ({
+  size,
+  imageSrc,
+  alt,
+  gravatarFallbackMd5,
+  organizationSlug,
```

Comment:
It's not supporting `...rest` props. `OrganizationAvatar` being a subtype of `Avatar` should accept all props of `Avatar`